library/manageiq_provider.py

```diff
@@ -470,6 +472,8 @@ def main():
                 endpoints.append(manageiq.generate_openshift_endpoint('hawkular', 'hawkular', h_hostname, h_port, token))
         elif provider_type == "amazon":
             endpoints = [manageiq.generate_amazon_endpoint('default', 'default', access_key_id, secret_access_key)]
+        elif provider_type == "hawkular-datawarehouse":
+            endpoints = [manageiq.generate_openshift_endpoint('default', 'default', hostname, port, token)]
```

Comment:
we should probably rename `generate_openshift_endpoint` to `generate_auth_key_endpoint` and `generate_amazon_endpoint` to `generate_user_password_endpoint`.
@cben what do you think?